Source/CrossingDetector/CrossingDetector.h

```diff
@@ -118,10 +131,11 @@ class CrossingDetector : public GenericProcessor
     bool negOn;
     int inputChan;
     int eventChan;
-    int shutoffChan; // temporary storage of chan w/ event that must be shut off; allows eventChan to be adjusted during acquisition
```

Comment:
Don't need this anymore since it's part of the event object that we save in turnoffEvent.